account_invoice_report_service/models/sale_order.py

```diff
@@ -19,3 +19,9 @@ def _compute_date_order_tz(self):
             datetime_utc = pytz.utc.localize(order.date_order)
             datetime_local = datetime_utc.astimezone(user_tz)
             order.date_order_tz = datetime_local.date()
+
+    service_only_date = fields.Date(compute="_compute_service_only_date")
+
+    def _compute_service_only_date(self):
+        for rec in self:
+            rec.service_only_date = fields.Date.context_today(rec, rec.service_date)
```

Comment:
```suggestion
    def _compute_service_date_tz(self):
        for rec in self:
            rec.service_date_tz = fields.Date.context_today(rec, rec.service_date)
```
Here we get the TZ from the current user, it's ok as we discussed